dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Models/OpenAIModel.cs

```diff
@@ -19,7 +19,10 @@
 using Azure.AI.OpenAI.Chat;
 using OpenAI.Chat;
 using Microsoft.Teams.AI.Application;
+using System.Reflection;
+using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Microsoft.Teams.AI.Tests")]
```

Comment:
just curious, what does this line do?

Reply:
it makes internal properties of the namespace visible to the "Microsoft.Teams.AI.Tests" project. In this case it's so that I can test the `OpenAIModel.SetMaxTokens` method.